### DISCUSSION TITLE
Offline install of WSLg on windows 10.

### DISCUSSION THREAD

**Discussion by glitchpanda** [microsoft/wslg#985] [Q&A] [answered]
Hey. 
I was wondering if anyone could help. I am trying to get WSLg installed on windows 10 offline. Anyone have any suggestions on how I can get this working? 

Kind regards,
Glitch

**Comment by Biswa96** [comment, marked as answer]
WSL comes with wslg feature built-in. It can be installed using the package from here https://github.com/microsoft/WSL/releases

**Comment by glitchpanda** [comment]
Really appreciated it. I was being an idiot. Thanks for the help.
